### PR TITLE
docs(info): remove old flags

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
docs

**Did you add tests for your changes?**
NA
**If relevant, did you update the documentation?**
YES
**Summary**
Remove old flags that are not supported now.

**Does this PR introduce a breaking change?**
NO

**Other information**

### CODE REVIEW

File: packages/info/README.md [hunk 1]
```diff
@@ -32,18 +32,6 @@ npx webpack info [options]
 
 _Not supported for config_
 
-#### Options
-
-| Flag                | Description                                                     | Type        |
-| ------------------- | --------------------------------------------------------------- | ----------- |
-| `--help`            | Show help                                                       | [ boolean ] |
-| `--version`         | Show version number of `webpack-cli`                            | [ boolean ] |
```
Comment (anshumanv) [inline packages/info/README.md:40]: help and version are still supported right?

Comment (evilebottnawi) [reply packages/info/README.md:40]: Yes, `help` and `version` still supported

**Comment by anshumanv** [comment]
One question - how does the user know from CLI that we support `help` and `version`?

**Comment by snitin315** [comment]
Need to add them here I guess.
https://github.com/webpack/webpack-cli/blob/c8ca87858b81e0c23e161d227558d2f0aeac003a/packages/webpack-cli/lib/utils/cli-flags.js#L88-L95

**Comment by anshumanv** [comment]
IIRC all commands support help and version, adding to list will cause a lot of duplication, we can just print it additionally during help, will check it out

**Pull request merged**